### PR TITLE
Feature/upload to power bi

### PR DESCRIPTION
**Pull request by Lance-Simeon**
Need to update branch pointers before merging

### CODE REVIEW

**Review by jeff-simeon — COMMENTED** [review]
how is this going to work in terms of backward compatibility with existing tenants?

**Comment by Lance-Simeon** [comment]
Existing tenants would need this yml file at the time the sync.yml is merged to master. This is because the sync.yml checkout step references the resources from this file.

**Comment by jeff-simeon** [comment]
> Existing tenants would need this yml file at the time the sync.yml is merged to master. This is because the sync.yml checkout step references the resources from this file.

We need to design this in a way that it is backward compatible so that when we update AzurePipelineTemplates, everyone doesn’t break

**Review by jeff-simeon — DISMISSED** [review]

File: Sync.yml [hunk 1]
```diff
@@ -37,7 +37,12 @@ resources:
   - repository: Templates
     type: github
     name: simeoncloud/AzurePipelineTemplates
-    ref: refs/heads/master
+    ref: refs/heads/feature/movePowerBILogicToNewFile
+    endpoint: simeoncloud
+  - repository: Reporting
+    type: github
+    name: simeoncloud/reporting
```
Comment (jeff-simeon) [inline Sync.yml:44]: why do you need this defined still?

Comment (jeff-simeon) [reply Sync.yml:44]: I would not expect this repo to have any changes for PowerBI

**Review by jeff-simeon — COMMENTED** [review]
also needed in Baseline

**Review by jeff-simeon — DISMISSED** [review]

**Review by jeff-simeon — DISMISSED** [review]

**Review by jeff-simeon — DISMISSED** [review]

**Review by jeff-simeon — COMMENTED** [review]
Make sure to change the branch before merge

**Review by jeff-simeon — APPROVED** [review]

**Review by david-simeon — APPROVED** [review]

**Pull request merged**